appdata.patch

```diff
@@ -0,0 +1,49 @@
+From fe6d6ac54134ed80be121952893bf4aaa29995a8 Mon Sep 17 00:00:00 2001
```

Comment:
Maybe merge the two patches? Probably not that important since the patch won't be required in the future

Reply:
I'd like to keep the commit log readable.